connector/src/main/java/org/geysermc/connector/network/translators/item/ItemTranslatorRegistry.java

```diff
@@ -40,25 +40,55 @@
 import com.github.steveice10.opennbt.tag.builtin.ShortTag;
 import com.github.steveice10.opennbt.tag.builtin.StringTag;
 import com.github.steveice10.opennbt.tag.builtin.Tag;
+import com.github.steveice10.packetlib.packet.Packet;
+import com.nukkitx.nbt.CompoundTagBuilder;
+import com.nukkitx.protocol.bedrock.BedrockPacket;
 import com.nukkitx.protocol.bedrock.data.ItemData;
 
 import org.geysermc.connector.GeyserConnector;
 
+import org.geysermc.connector.network.session.GeyserSession;
+import org.geysermc.connector.network.translators.*;
 import org.geysermc.connector.utils.MessageUtils;
 import org.geysermc.connector.utils.Toolbox;
+import org.reflections.Reflections;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+import java.util.stream.Collectors;
 
-public class ItemTranslator {
+public class ItemTranslatorRegistry {
```

Comment:
As it seems there is more going on in this class than just registration, I feel like this naming should be changed back.